model-optimizer/mo/ops/op.py

```diff
@@ -458,3 +462,42 @@ def get_nchw_to_nhwc_permutation(dims_number: int):
             perm = list(range(0, dims_number))
         inv = PermuteAttrs.get_inverse_permutation(perm)
         return PermuteAttrs.Permutation(perm=int64_array(perm), inv=int64_array(inv))
+
+
+class RTInfo:
```

Comment:
This doesn't look extendable -- how would we add other rt_infos to that?

Reply:
Refactored RTInfo as we discussed. Please take a look.